### DISCUSSION TITLE
The idle notifier does not work at the blisterwood tree.

### DISCUSSION THREAD

**Discussion by Anzelinio** [runelite/runelite#15827] [Q&A] [answered]
I tried all settings but there is no notification once you stop chopping logs.

**Comment by Nightfirecat** [comment, marked as answer]
This is fixed in master in commit 12da1231c566fc0c29630bca77f0bb1dabea433d and will be fixed in our next release.

**Reply by Anzelinio** [reply]
You are the best. Thank you.
